File: client/src/lib/equipment/sse.ts
export enum EquipmentServerEvent {
	EquipmentCreate = "equipment:create",
	EquipmentAnomaly = "equipment:anomaly",
	EquipmentReallocate = "equipment:reallocate",

	BorrowRequestCreate = "borrow-request:create",
	BorrowRequestUpdate = "borrow-request:update",
	BorrowRequestReview = "borrow-request:review",

	ReturnRequestConfirm = "return-request:confirm",
}

File: server/equipment/sse.go
package equipment

type event = string

const (
	eventEquipmentCreate     event = "equipment:create"
	eventEquipmentAnomaly    event = "equipment:anomaly"
	eventEquipmentReallocate event = "equipment:reallocate"
)

const (
	eventBorrowRequestCreate event = "borrow-request:create"
	eventBorrowRequestUpdate event = "borrow-request:update"
	eventBorrowRequestReview event = "borrow-request:review"
)

const (
	eventReturnRequestConfirm event = "return-request:confirm"
)
